src/services/nasc/routes/ac.ts

```diff
@@ -89,12 +91,12 @@ async function processLoginRequest(server: HydratedServerDocument, pid: number,
 
 async function processServiceTokenRequest(server: HydratedServerDocument, pid: number, titleID: string): Promise<URLSearchParams> {
 	const tokenOptions = {
-		system_type: 0x2, // * 3DS
-		token_type: 0x4, // * Service token
+		system_type: SystemType.CTR,
+		token_type: TokenType.IndependentService,
 		pid: pid,
 		access_level: 0,
 		title_id: BigInt(parseInt(titleID, 16)),
-		expire_time: BigInt(Date.now() + (3600 * 1000))
+		expire_time: BigInt(Date.now()) // TODO - Hack. Independent services expire their own tokens, so we give them the ISSUED time, not an EXPIRE time
```

Comment:
I think it would be nice if service tokens had a different TS type. That's what we did on the Juxt end (since we never decrypt OAuth tokens there).

Reply:
With the impending token updates that will happen in the future, is it worth doing more than a hack?

This is somewhat related to a point I mentioned in the PRs opening comment though. Maybe we should just switch gears and give every token a "issued time" instead of an "expired time", for consistency. We would just need to change the expiration check a bit but that's not terribly hard

Reply:
The issue with changing other token types is that there's an expiration check in nex-go, so we'd need to first update every NEX app that does token checks.